### PR TITLE
Add Command Palette Commands

### LINKED ISSUE #11: Support for Meta-Shift-P menu
I would like to be able to find and use the commands from the <kbd>cmd</kbd>+<kbd>shift</kbd>+<kbd>P</kbd> menu (<kbd>ctrl</kbd>+<kbd>shift</kbd>+<kbd>P</kbd> in Windows).

### PR DESCRIPTION
**Pull request by KyleKing**
Fixes #11

Tested locally and looks like it works!

### CODE REVIEW

File: GitLink.sublime-commands [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    },
+    {
+        "caption": "GitLink: Open Git Blame URL to file",
+        "command": "gitlink",
+        "args": {
+            "line": false,
+            "web": true,
+            "blame": true
+        }
+    }
```
Comment (KyleKing) [inline GitLink.sublime-commands:64]: I think I have all of the possible combinations, but do you know what the most common shortcuts are? We could add only the "Open" variations and add instructions to the README for how users can customize the command palette (add snippets in a new `GitLink.sublime-commands` in their `User/` directory)

File: GitLink.sublime-commands [hunk 2]
```diff
@@ -0,0 +1,48 @@
+[
+    {
+        "caption": "GitLink: Copy Git URL to file",
+        "command": "gitlink",
+        "args": {
+            "line": false
+        }
+    },
```
Comment (KyleKing) [inline GitLink.sublime-commands:8]: `sublime.set_clipboard()` doesn't appear to be modifying my clipboard. Is that a know issue? I can open a new ticket and troubleshoot later

Comment (rscherf) [reply GitLink.sublime-commands:8]: To be honest, I'm not exactly sure.

File: GitLink.sublime-commands (hunk 1 above)
Comment (rscherf) [reply GitLink.sublime-commands:64]: I imagine we'd also want menu items for navigating directly to the line as well, no?

Comment (KyleKing) [reply GitLink.sublime-commands]: Done! https://github.com/rscherf/GitLink/pull/24/commits/1af1c185b3c446e8902d11dd4562f49c46c99964

File: GitLink.sublime-commands (hunk 2 above)
Comment (KyleKing) [reply GitLink.sublime-commands]: Found the problem, the script was failing here: https://github.com/rscherf/GitLink/pull/24/commits/e3daa8aa2ecc86d33506af02395e642ea21322a9

**Pull request merged**